src/OpenSpaceToolkit/Astrodynamics/EventCondition/RealCondition.cpp

```diff
@@ -90,6 +93,19 @@ String RealCondition::StringFromCriterion(const Criterion& aCriterion)
     return String::Empty();
 }
 
+RealCondition RealCondition::DurationCondition(const Criterion& aCriterion, const Duration& aDuration)
+{
+    return {
+        "Duration",
+        aCriterion,
+        [](const State& aState) -> Real
+        {
+            return (aState.accessInstant() - Instant::J2000()).inSeconds();
+        },
+        {aDuration.inSeconds(), RealCondition::Target::Type::RelativeSegmentStart}
```

Comment:
```suggestion
        {aDuration.inSeconds(), EventCondition::Target::Type::RelativeSegmentStart}
```

nit: seems a little clearer to me to use the original base class in which it's defined. Don't feel strongly about this one though